fastlane_core/spec/helper_spec.rb

```diff
@@ -52,6 +52,11 @@
         expect(FastlaneCore::Helper.ci?).to be(true)
       end
 
+      it "returns true when building in AppCenter" do
+        stub_const('ENV', { 'APPCENTER_OUTPUT_DIRECTORY' => '/Users/vsts/agent/2.133.3/work/1/a/build' })
```

Comment:
I think it's not a good idea to hardcode local paths

Reply:
What do you suggest? The idea is to check if a value exists for `APPCENTER_OUTPUT_DIRECTORY`.

Reply:
Oh, ok, I thought it's actually doing something with the directory at that path. Maybe you could replace it with `/path/to/a/build` to avoid confusion?

Reply:
sure thing.

Reply:
done.